source/uwp/Renderer/lib/XamlBuilder.cpp

```diff
@@ -1251,72 +1251,76 @@ namespace AdaptiveNamespace
                         .Get(),
                     &clickToken));
 
-                ABI::AdaptiveNamespace::Sentiment actionSentiment;
-                THROW_IF_FAILED(action->get_Sentiment(&actionSentiment));
+                HString actionSentiment;
+                THROW_IF_FAILED(action->get_Sentiment(actionSentiment.GetAddressOf()));
 
-                if (actionSentiment == ABI::AdaptiveNamespace::Sentiment_Positive ||
-                    actionSentiment == ABI::AdaptiveNamespace::Sentiment_Destructive)
-                {
-                    ComPtr<IResourceDictionary> resourceDictionary;
-                    THROW_IF_FAILED(renderContext->get_OverrideStyles(&resourceDictionary));
-                    ComPtr<IInspectable> subtleOpacityInspectable;
+                INT32 isSentimentPositive{}, isSentimentDestructive{}, isSentimentDefault{};
+
+                THROW_IF_FAILED(WindowsCompareStringOrdinal(actionSentiment.Get(), HStringReference(L"positive").Get(), &isSentimentPositive));
+                THROW_IF_FAILED(WindowsCompareStringOrdinal(actionSentiment.Get(), HStringReference(L"destructive").Get(), &isSentimentDestructive));
```

Comment:
does this comparator ignore cases? and is ignoring cases better approach?

Reply:
I tested if upper case or lower case would affect anything and it didnt, also most properties use lower case values so it makes sense for this one to be lower cased too, right?